lib/fs.js

```diff
@@ -217,7 +231,7 @@ function access(path, mode, callback) {
     mode = F_OK;
   }
 
-  path = getValidatedPath(path);
+  path = possiblyTransformPath(getValidatedPath(path));
```

Comment:
Perhaps the possiblyTransformPath should be moved into getValidatedPath given that they'll likely always need to be used together